zerver/views/custom_profile_fields.py

```diff
@@ -27,10 +27,17 @@ def list_realm_custom_profile_fields(request: HttpRequest, user_profile: UserPro
 @has_request_variables
 def create_realm_custom_profile_field(request: HttpRequest,
                                       user_profile: UserProfile, name: Text=REQ(),
+                                      hint: Optional[Text]=REQ(default=None),
```

Comment:
@timabbott, is there any reason why `REQ(default=None)` doesn't work here? I am getting this error:
> REQ doesn't return a value

Reply:
Yeah, I thought so too, but the only thing that works is `REQ(default='')`.

Reply:
(Empty string serves our purpose as well.)